sheetstorm/settings/base.py

```diff
@@ -232,6 +232,7 @@
 STATIC_URL = '/static/'
 
 AUTH_USER_MODEL = 'users.CustomUser'
+LOGIN_URL = "users/accounts/login"
```

Comment:
this seems to be unused

Reply:
This is not unused. This is used by django auth and this is using `settings.LOGIN_URL` which normally is hardcoded for "accounts/login"

Reply:
Still please use `reverse` or `reverse_lazy`.

Reply:
Oh, I forgot about `reverse_lazy`. Done